manifest.json

```diff
@@ -167,29 +167,29 @@
         "files": ["angular-mocks.js", "angular-scenario.js"]
       },
       "angularTranslate": {
-        "version": "2.8.1",
+        "version": "2.18.1",
         "downloadFormat": "zip",
-        "url": "https://github.com/angular-translate/bower-angular-translate/archive/2.8.1.zip",
+        "url": "https://github.com/angular-translate/bower-angular-translate/archive/2.18.1.zip",
```

Comment:
@apb7, Can we use[ cdnjs](https://cdnjs.com/libraries/angular-translate) and only download the files which are required?  (not sure!)
Asking this:
- As downloading files through CDNs use to be fast and we can download specific files which will be less in size.

Reply:
yes we can have a try probably with lightweight static libraries

Reply:
Hi @DubeySandeep, I've added the cdnjs for the angular static libs PTAL Thanks!